### DISCUSSION TITLE
prometheus metrics: network_chain_tip_height_estimator

### DISCUSSION THREAD

**Discussion by jleni** [ZcashFoundation/zebra#7962] [General]
Is there any way to access the "estimated chain tip height" as a prometheus metrics?

**Comment by teor2345** [comment]
We have it as a progress bar, but we haven't added it as a metric:
https://github.com/ZcashFoundation/zebra/blob/3be22b2cdaa59cd1c028432764f718a3cbef370d/zebrad/src/components/sync/progress.rs#L146-L156

Is that something you'd like us to add, or is it code you're comfortable submitting a PR for?

Here's an example of what a metric looks like in Rust:
https://github.com/ZcashFoundation/zebra/blob/3be22b2cdaa59cd1c028432764f718a3cbef370d/zebra-state/src/service/queued_blocks.rs#L374

We could call it `state.estimated.chain.tip.height`?

**Comment by mpguerra** [comment]
closing this one as stale
